services/opsgenie/service.go

```diff
@@ -0,0 +1,125 @@
+package opsgenie
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/influxdb/kapacitor"
+)
+
+type Service struct {
+	apikey       string
+	teams        string
+	recipients   string
+	url          string
+	recovery_url string
+	global       bool
+	logger       *log.Logger
+}
+
+func NewService(c Config, l *log.Logger) *Service {
+	return &Service{
+		teams:        c.Teams,
+		recipients:   c.Recipients,
+		apikey:       c.APIKey,
+		url:          c.URL + "/",
+		recovery_url: c.RecoveryURL + "/",
+		global:       c.Global,
+		logger:       l,
+	}
+}
+
+func (s *Service) Open() error {
+	return nil
+}
+
+func (s *Service) Close() error {
+	return nil
+}
+
+func (s *Service) Global() bool {
+	return s.global
+}
+
+func (s *Service) Alert(teams, recipients, messageType, message, entityID string, t time.Time, details interface{}) error {
```

Comment:
This will need to change as well:

```
func (s *Service) Alert(teams, recipients []string , messageType, message, entityID string, t time.Time, details interface{}) error {
```